pkg/common/constants.go

```diff
@@ -83,7 +83,7 @@ const (
 	FailedWarnThreshold = 0.3
 	// FailedTerminateThreshold Terminate experiment if the percentage of failed invocations (e.g., connection timeouts,
 	// function timeouts) is greater than this threshold
-	FailedTerminateThreshold = 0.5
+	FailedTerminateThreshold = 1.1
```

Comment:
Why do you change it here?

Reply:
I changed this for testing, but I was not sure why we want the drivers to terminate if the failure rate is over 50%. Shouldn't it be okay to run them even if the rate of failure is above 50%?